public/pages/patternlibrary.html

```diff
@@ -475,6 +476,7 @@ <h2>Molecules</h2>
             text-decoration: none;
             padding-bottom: 5px;
             color: #333;
+            white-space: nowrap;
```

Comment:
this should have been included in the other PR, it shouldn't be in this PR.

Reply:
I am confused by this. You already made this change here: https://github.com/team-refcode/refcode.org/pull/291/files#diff-6e1847c0e3f18da0864c474313b27fbd4c7be499c086f520e05fa88bf93781f6R478

The fact that it's showing up here as an addition is very suspicious. What's going on here?